.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+AllCops:
+  Include:
+    - 'lib/statsample-glm/glm/formula.rb'
+    - 'lib/statsample-glm/glm/regression.rb'
```

Comment:
Unfortunately, as I checked it, there seems to be no easy way to "include only this and that file". However, there are two solution:

1. Readable yet non-DRY: just list ALL the files in exclude (except your two ones)
2. Smart yet not very readable: use in `Exclude` regexp with negative lookbehind, like this:
```yaml
Exclude:
  - !ruby/regexp /(?<!formula\.rb|regression\.rb)$/
```
It reads like "matches end-of-line" (in fact, any line therefore), UNLESS it was this or that filename before the matched end-of-line.
If you'll prefer the latter solution, please leave the comment in `.rubocop.yml`, explaining it.